.pre-commit-config.yaml

```
@@ -20,7 +20,7 @@ repos:
           - methods
           - rlang
           - stats
-          - teal.code
+          - insightsengineering/teal.code
```

I think you need to revert this

```suggestion
          - teal.code
```

---

Ok, I'll start a PR to re-add this and discuss it.

`pre-commit` is a developer tool and it fails to build documentation when there are docs dependencies between packages (for example `@inheritParams`)